db/migrate/20210611120159_add_git_important_files_hash_to_solutions_and_exercises.rb

```diff
@@ -0,0 +1,17 @@
+class AddGitImportantFilesHashToSolutionsAndExercises < ActiveRecord::Migration[6.1]
+  def change
+    add_column :solutions, :git_important_files_hash, :string, null: true
+    add_column :exercises, :git_important_files_hash, :string, null: true
+
+    Exercise.find_each do |exercise|
+      exercise.update!(git_important_files_hash: Git::GenerateHashForImportantExerciseFiles.(exercise))      
+    end
+
+    Solution.find_each do |solution|
+      solution.update!(git_important_files_hash: exercise.git_important_files_hash)      
```

Comment:
We should also update the `git_sha` and `git_slug` to match the exercise here (ie we're updating everything). Else the two values will be potentially mismatched.